test/e2e_clutz_dts_test.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// tslint:disable:no-unused-expression mocha .to.be.empty getters.
+
+import {expect} from 'chai';
+import * as ts from 'typescript';
+
+import * as tsickle from '../src/tsickle';
+
+import {compileWithTransfromer, dtsCompilerOptions} from './test_support';
+
+describe('clutz dts', () => {
+  it('does nothing if there are no exports', () => {
+    const sources = new Map<string, string>();
+    sources.set('input.ts', `const x = 1;`);
+
+    // Run tsickle+TSC to convert inputs to Closure JS files.
+    const {files} = compileWithTransfromer(sources, dtsCompilerOptions);
+    const dts = files.get('input.d.ts')!;
+
+    expect(dts).not.to.contain('namespace ಠ_ಠ.clutz');
+  });
+
```

Comment:
add test for CJS style export - 'let x = 0; export = x;`

Reply:
Done.